source/blog/2018-11-19-ember-3-6-released.md

```diff
@@ -0,0 +1,209 @@
+For help on how to do this please refer to the [deprecation app](https://deprecations-app-prod.herokuapp.com/deprecations/v3.x/#toc_remove-handler-infos).
+
+**Deprecate Router Events (5 of 6)**
+
+Currently, application-wide transition monitoring is spread out throughout the `Route` classes. This does not really belong here but in the `Router` service instead.
+
+<!--alex ignore hooks-->
+That is the reason for the existing `willTransition` and `didTransition` hooks/events on the Router. But they are not sufficient to capture all the detail people need.
+
+In addition, they receive `handlerInfos` in their arguments, which are an undocumented internal implementation detail of router.js that doesn't belong in Ember's public API. Everything you can do with handlerInfos can be done with the `RouteInfo`.
+
+For examples on how to transition both the `Route` and `Router` usages of `willTransition` and `didTransition`, please refer to the [deprecation app](https://deprecations-app-prod.herokuapp.com/deprecations/v3.x/#toc_deprecate-router-events).
+
+**Transition State Removal (6 of 6)**
+
+The `Transition` object is a public interface that actually exposed internal state used by router.js to perform routing.
+
+Accessing `state`, `queryParams` or `params` on the `Transition` has been removed. If you need access to information about the routes, you are probably better served injecting the router service as it exposes a publically supported version of the `RouteInfo`s.
+
+For information on how to do this please refer to the [deprecation app](https://deprecations-app-prod.herokuapp.com/deprecations/v3.x/#toc_transition-state).
+
+---
+
+## Ember Data
+
+Ember Data is the official data persistence library for Ember.js applications.
+
+### Changes in Ember Data 3.6
+
+#### New Features (0)
+
+No new features introduced in Ember Data 3.6.
+
+#### Deprecations (0)
+
+No new deprecations introduced in Ember Data 3.6.
+
+
+For more details on changes in Ember Data VER, please review the
+[Ember Data 3.6 release page](https://github.com/emberjs/data/releases/tag/v3.6).
+
+---
+
+## Ember CLI
+
+Ember CLI is the command line interface for managing and packaging Ember.js applications.
+
+### Upgrading Ember CLI
+
+You may upgrade Ember CLI separately from Ember.js and Ember Data! To upgrade your projects using `yarn` run:
+
+```bash
+yarn upgrade ember-cli
+```
+
+To upgrade your projects using `npm` run:
+
+```bash
+npm install --save-dev ember-cli
+```
+
+After running the upgrade command, make sure to install (if you haven't already) ember-cli-update globally:
+
+```bash
+npm install -g ember-cli-update
+```
+
+This utility will help you to update your app or add-on to the latest Ember CLI version. You will probably encounter merge conflicts, in which the default behavior is to let you resolve conflicts on your own.
```

Comment:
I'm not sure those instructions are correct. @kellyselden can you check this?

Reply:
That would be great to know. These instructions are in our release post template and were also published in previous release posts.